frontend/app/src/next/pages/authenticated/dashboard/workers/managed/components/managed-worker-build-logs.tsx

```diff
@@ -0,0 +1,28 @@
+import { queries } from '@/lib/api';
+import { useQuery } from '@tanstack/react-query';
+import LoggingComponent from '@/components/cloud/logging/logs';
+
+export function ManagedWorkerBuildLogs({ buildId }: { buildId: string }) {
+  const getBuildLogsQuery = useQuery({
+    ...queries.cloud.getBuildLogs(buildId),
+    refetchInterval: 5000,
+  });
+
+  const logs = getBuildLogsQuery.data?.rows || [
+    {
+      line: 'Loading...',
+      timestamp: new Date().toISOString(),
```

Comment:
I think this `new Date` needs to be in a `useState` or `useRef` probably so it doesn't cause infinite renders

Reply:
(not 100% sure of this)